Makefile

```diff
@@ -218,14 +218,24 @@ docker-build-all: docker-build-debug docker-build-hermes
 ###############################################################################
 ###                                Linting                                  ###
 ###############################################################################
+golangci_lint_cmd=golangci-lint
+golangci_version=v1.50.1
 
 lint:
 	@echo "--> Running linter"
-	@go run github.com/golangci/golangci-lint/cmd/golangci-lint run --timeout=10m
+	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(golangci_version)
+	@$(golangci_lint_cmd) run --timeout=10m
+
+lint-fix:
```

Comment:
What's the difference between `make lint-fix` and `make format`?

Reply:
`make lint-fix` is just running the golanci linter, while `make format` is running `gofumpt` on everything that's not excluded by `-not -path`, and then it runs golanci linter afterwards.